testing/integration/run.sh

```
@@ -162,7 +189,7 @@ fi
 function cleanup() {
     local exit_code=$1
     printf "Cleanup before exit..."
-    pkill valkey-server || true
+    pkill -9 valkey-server || true
     deactivate >/dev/null 2>&1 || true
     cd ${ROOT_DIR}
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)